x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/datafeed/DatafeedConfigTests.java

```diff
@@ -100,7 +113,7 @@ public static DatafeedConfig createRandomizedDatafeedConfig(String jobId, long b
             if (aggHistogramInterval == null) {
                 builder.setFrequency(TimeValue.timeValueSeconds(randomIntBetween(1, 1_000_000)));
             } else {
-                builder.setFrequency(TimeValue.timeValueMillis(randomIntBetween(1, 5) * aggHistogramInterval));
+                builder.setFrequency(TimeValue.timeValueSeconds(randomIntBetween(1, 5) * aggHistogramInterval));
```

Comment:
This test failed with the error:
```
java.lang.IllegalArgumentException: frequency has to be a multiple of 1s; actual was '900ms'
...
   >    at org.elasticsearch.xpack.core.ml.utils.time.TimeUtils.checkMultiple(TimeUtils.java:101)
   >    at org.elasticsearch.xpack.core.ml.utils.time.TimeUtils.checkPositiveMultiple(TimeUtils.java:74)
```

It's clear that `TimeValue.timeValueMillis(randomIntBetween(1, 5) * aggHistogramInterval)` is unlikely to return a whole number of seconds but I'm amazed it has not happened before.

Reproduce command was
```
/gradlew :x-pack:plugin:core:test -Dtests.seed=7D58D672CDD2EF69 -Dtests.class=org.elasticsearch.xpack.core.ml.datafeed.DatafeedConfigTests -Dtests.method="testToXContentForInternalStorage" -Dtests.security.manager=true -Dtests.locale=ln -Dtests.timezone=America/Rankin_Inlet -Dcompiler.java=10 -Druntime.java=10
```

I'll make the same change in `...protocol...DatafeedConfigTests`

Reply:
Actually there is no validation in `protocol..DatafeedConfigTests` so not an issue